tools/ConsoleTypes.natvis

```diff
@@ -107,10 +107,9 @@
     </Type>
 
     <Type Name="til::generational&lt;*&gt;">
-        <DisplayString>{{ gen={_generation._value}, {_value} }}</DisplayString>
-        <Expand>
-            <ExpandedItem>_value</ExpandedItem>
-        </Expand>
+        <AlternativeType Name="$T1" />

```

Comment:
Oh, is this an even better NatVis?

Reply:
Yes, I couldn't figure out how to make `<SmartPointer>` work, so that the inner value of `til::generational` is visible in the debugger. As it turns out, the `<AlternativeType>` is required for `<SmartPointer>` to work. This isn't documented anywhere unfortunately (nor does it make any sense frankly, since the debugger should know what the result type of the expression in `<SmartPointer>` is).